multi-int-store/src/main/java/com/connexta/multiintstore/common/StorageManager.java

```diff
@@ -24,13 +24,13 @@
   private static final String CST = "cst";
```

Comment:
❓ Should this variable name change now that we call it `IndexedProductMetadata`?